interfaces/IDeviceInfo.h

```diff
@@ -25,136 +25,315 @@
+            IDeviceVideoCapabilities::ScreenResolution outputResolutions /* @bitmask @brief Supported resolutions */;
+            IDeviceVideoCapabilities::ScreenResolution defaultResolution /* @brief Default resolution */;
+        };
+
+        using IVideoOutputCapsIterator = RPC::IIteratorType<VideoOutputCaps, ID_DEVICE_CAPABILITIES_VIDEO_OUTPUT>;
+
+        // @alt supportedvideodisplays
+        // @property
+        // @brief Retrieves VideoOutputs
+        virtual Core::hresult VideoOutputs(VideoOutput& videoOutputs /* @out @bitmask */) const = 0;
+        // @brief Retrieves DefaultResolution against given video Output
+        virtual Core::hresult DefaultResolution(const VideoOutput videoOutput /* @index */, ScreenResolution& defaultResolution /* @out */) const = 0;
+        // @alt supportedresolutions
+        // @brief Retrieves Resolution against given video Output
+        virtual Core::hresult Resolutions(const VideoOutput videoOutput /* @index */, ScreenResolution& resolutions /* @out @bitmask */) const = 0;
+        // @alt supportedhdcp
+        // @brief Retrieves Hdcp
+        virtual Core::hresult Hdcp(const VideoOutput videoOutput /* @index */, CopyProtection& hdcpVersion /* @out */) const = 0;
+        // @property
+        // @brief Retrieves Host EDID
+        virtual Core::hresult HostEDID(string& edid /* @out */) const = 0;
+        // @property
+        // @brief Retrieves HDR
+        virtual Core::hresult HDR(bool& supportsHDR /*@out*/) const = 0;
+        // @property
+        // @brief Retrieves Atoms
+        virtual Core::hresult Atmos(bool& supportsAtmos /*@out*/) const = 0;
+        // @property
+        // @brief Retrieves cec
+        virtual Core::hresult CEC(bool& supportsCEC /*@out*/) const = 0;
+
+        // @brief Retrieves DeviceVideoCapabilities
+        virtual Core::hresult DeviceVideoCapabilities(string& edid /* @out */,
+            bool& hdr /* @out */, bool& atmos /* @out */, bool& cec /* @out */,
+            IVideoOutputCapsIterator*& videoOutputCaps /* @out */) const = 0;
+    };
+
+    /* @json 1.0.0 */
+    struct IDeviceMetadata : virtual Core::IUnknown {
+        enum { ID = ID_DEVICE_METADATA };
+
+        virtual ~IDeviceMetadata() override = default;
+
+        struct Device {
+            string deviceType /* @brief Device type */;
+            string distributorId /* @brief Partner ID or distributor ID for device */;
+            string friendlyName /* @brief Friendly name */;
+            string make /* @brief Device manufacturer */;
+            string modelName /* @brief Model Name */;
+            uint16_t modelYear /* @brief Model Year */;
+            string platformName /* @brief Platform name */;
+            string serialNumber /* Device serial number */;
+            string sku /* Device model number or SKU */;
+        };
+
+        // @property
+        // @brief Retrieves Device Info
+        virtual Core::hresult DeviceData(Device& device /* @out */) const = 0;
```

Comment:
Maybe for consistency, if possible, call these methods like the interface bu than without the I, so:
virtual Core::hresult DeviceMetadata(Device& device /* @out */) const = 0;